<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalAccountWindow</class>
 <widget class="QWidget" name="PersonalAccountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>474</width>
    <height>693</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>474</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  qlineargradient( spread:pad,  x1:0, y1:0, x2:1, y2:0,  stop:0 rgba(105, 57, 153, 100),  stop:1 rgba(165, 105, 189, 255));  border: 1px solid #5D3B9B;  border-radius: 5px;
font-family: Everlasting;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>341</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">backgroud-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;</string>
   </property>
   <widget class="QLineEdit" name="NewAddres">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Return_button_Clicked">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>650</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 12px;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Вернуться</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewPassword">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="oldPassword">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewPassword2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
border: none;
font-size: 12pt;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Личный кабинет</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newPhone">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>13</number>
    </property>
   </widget>
   <widget class="QLabel" name="FMLName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Тут типо ФИО</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Адрес:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Телефон:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 12pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Смена пароля:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Старый пароль:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Новый адрес:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Подтверждение:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Adres">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>тут типо адрес</string>
    </property>
   </widget>
   <widget class="QLabel" name="Phone">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>тут типо телефон</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>новый пароль:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 12pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Смена адреса:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangePass_button_Clicked">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 12px;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Сменить  пароль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeNumber_button_Clicked">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>600</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 12px;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Сменить телефон</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 12pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Смена телефона:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeAddres_button_clicked">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 12px;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Сменить адрес</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Новый номер:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PassInfo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>тут типо телефон</string>
    </property>
   </widget>
   <widget class="QLabel" name="AddresInfo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>тут типо телефон</string>
    </property>
   </widget>
   <widget class="QLabel" name="PhoneInfo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>тут типо телефон</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
